packages/model-viewer/src/features/animation.ts

```diff
@@ -13,29 +13,57 @@
  * limitations under the License.
  */
 
-import {property} from 'lit/decorators.js';
-import {LoopOnce, LoopPingPong, LoopRepeat} from 'three';
+import { property } from 'lit/decorators.js';
```

Comment:
It looks like your formatter isn't picking up our `.clang-format` file? Can you get it to keep consistent formatting with our files?